CHANGELOG-zh-CN.md

```diff
@@ -1,5 +1,108 @@
 # OceanBase Developer Center (ODC) CHANGELOG
 
+## 4.2.3 (2023-12-15)
+
+### 功能变化
+
+导入导出
+
+- 支持原生 MySQL 数据源的导入导出
+
+数据库对象管理
+
+- OceanBase MySQL 以及原生 MySQL 模式下支持 GIS 数据类型
+
+数据源
+
+- 允许数据源绑定到项目
+
+命令行窗口
+
+- 允许自定义禁用命令
```

Comment:
we not offer this feature to odc user, it is an obclient feature, odc just use it

Reply:
okay, delete it~